src/builders.ts

```diff
@@ -122,20 +122,31 @@ export async function broadcastTransaction(
   transaction: StacksTransaction,
   network: StacksNetwork
 ): Promise<string> {
-  const tx = transaction.serialize();
+  const rawTx = transaction.serialize();
+  const url = network.getBroadcastApiUrl();
+
+  return broadcastRawTransaction(rawTx, url);
+}
 
+/**
+ * Broadcast the signed transaction to a core node
+ *
+ * @param {Buffer} rawTx - the raw serialized transaction buffer to broadcast
+ * @param {string} url - the broadcast endpoint URL
+ *
+ * @returns {Promise} that resolves to a response if the operation succeeds
+ */
+export async function broadcastRawTransaction(rawTx: Buffer, url: string) {
```

Comment:
@yknl if I am passing a string, is it technically of type `Buffer`?

Reply:
or would I need to do `broadcastRawTransaction(Buffer.from(rawTxString), url)`

Reply:
yes you would need to convert it to a Buffer first

Reply:
cool, thanks for clarifying